.env.template

```diff
@@ -170,4 +170,4 @@ ELEVENLABS_VOICE_2_ID=your-voice-id-2
 TW_CONSUMER_KEY=
 TW_CONSUMER_SECRET=
 TW_ACCESS_TOKEN=
-TW_ACCESS_TOKEN_SECRET=
+TW_ACCESS_TOKEN_SECRET=
```

Comment:
Please entirely revert change to this file. Should not be included in the PR.

Reply:
Thank you, but I just added send_telegram functionality which required making changes to .env.template